pt_miniscreen/app.py

```diff
@@ -96,11 +100,23 @@ def handle_down_button_release(self):
         self.root.scroll_down()
 
     def display(self):
+        def restore_miniscreen():
```

Comment:
This should be put on the class to avoid recreating it on every display, which can be called very frequently.